src/controllers/controller.ts

```diff
@@ -205,4 +205,28 @@ export default class Controller {
     }
     res.sendStatus(200);
   };
+
+  public newProjectTelemetry = (req: Request, res: Response) => {
+    const payload: JwtPayload | false = this.authorize(req.headers.authorization, res);
+    if (payload === false) return;
+    if (payload.permission !== 'Szakember') return res.sendStatus(403);
+
+    mssql.query`INSERT INTO Telemetry(projectID, stateID, startDate, finishDate) VALUES(${req.body.projectID}, ${req.body.stateID},  GETDATE(), NULL);`
+      .then(() => {
+        res.sendStatus(201);
+      })
+      .catch((err) => res.status(400).send({ msg: err as string }));
+  };
+
+  public getProjects = (req: Request, res: Response) => {
+    const payload: JwtPayload | false = this.authorize(req.headers.authorization, res);
+    if (payload === false) return;
+    if (payload.permission !== 'Szakember') return res.sendStatus(403);
+
+    mssql.query`SELECT * FROM Projects p JOIN Telemetry t ON p.projectID = t.projectID;`
+      .then((result) => {
+        res.status(200).send({ result: result.recordset });
```

Comment:
```suggestion
        res.status(200).send(result.recordset);
```
elég csak a tömböt visszaadni külön adattag nélkül, majd 'getAllParts'-nál is javítani kell(utóbbihoz a frontendbe is bele kell nyúlni)